package.json

```diff
@@ -116,6 +116,7 @@
     "json-loader": "^0.5.7",
     "lodash": "^4.9.0",
     "node-sass": "4.9.0",
+    "markdown-it": "~8.4.1",
```

Comment:
Let's pin this to the exact version number, in case there's semver principles are skipped upstream :)